javascript/src/main/App.js

```diff
@@ -14,12 +14,14 @@ import SlackUsers from "./pages/Admin/SlackUsers";
 import SearchInfo from "./pages/Admin/SearchInfo";
 import AuthorizedRoute from "main/components/Nav/AuthorizedRoute";
 import ChannelList from "main/pages/Channels/ChannelList";
-
 import ChannelPageList from "./pages/Channels/ChannelPageList";
 import ChannelPageScrollable from "./pages/Channels/ChannelPageScrollable";
-
+import AnalyzeReactions from "main/pages/AnalyzeMessageData/AnalyzeReactions";
+import CountMessagesByUser from "main/pages/AnalyzeMessageData/CountMessagesByUser";
+import HistogramOfMessagesByUser from "main/pages/AnalyzeMessageData/HistogramOfMessagesByUser";
+import SearchMessagesByUser from "main/pages/AnalyzeMessageData/SearchMessagesByUser";
 import Search from "main/pages/Search/Search";
-
+import ChannelPage from "./pages/Channels/ChannelPage";
```

Comment:
can you change this to an absolute path import? (i know it isn't consistent with the line directly below it, but we want to avoid relative path imports)